pandas/tests/frame/test_reductions.py

```diff
@@ -1068,13 +1068,17 @@ def test_idxmax_dt64_multicolumn_axis1(self):
 
     @pytest.mark.parametrize("opname", ["any", "all"])
     def test_any_all(self, opname, bool_frame_with_na, float_string_frame):
-        assert_bool_op_calc(
-            opname, getattr(np, opname), bool_frame_with_na, has_skipna=True
-        )
         assert_bool_op_api(
             opname, bool_frame_with_na, float_string_frame, has_bool_only=True
         )
 
+    @pytest.mark.xfail(reason="GH12863: numpy result won't match for object type")
+    @pytest.mark.parametrize("opname", ["any", "all"])
```

Comment:
so this is trying to match numpy? do we need this anymore?

Reply:
I doubt the original purpose was matching numpy, probably more for adding tests without having to hardcode an expected result. For now I modified this slightly to remove the need for the xfail, but probably could be removed (there's coverage, but it's scattered).